ext/session/mod_user.c

```diff
@@ -75,7 +75,15 @@ PS_OPEN_FUNC(user)
 	zval args[2];
 	STDVARS;
 
+	if (PS(in_save_handler)) {
+		PS(in_save_handler) = 0;
+		php_error_docref(NULL, E_WARNING, "Cannot call save handler function recursive manner");
```

Comment:
Thanks for feedback! I'll update error message to
Cannot call session save handler in a recursive manner
because invalid calls do not have to be recursive call.